core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.kt

```diff
@@ -109,6 +110,23 @@ abstract class CoreMainActivity : BaseActivity(), WebViewProvider {
     if (destination.id !in topLevelDestinations) {
       handleDrawerOnNavigation()
     }
+    if (destination.id == readerFragmentResId) {
+      // show reader nav view
+      showReaderNavView()
```

Comment:
the comment is unnecessary when it is literally identical to the method name but the larger problem with these method names is that they are untrue, this does not show the  Reader Table of Contents Drawer, it unlocks it. The hide is only partially true.